docs/proposals/federation.md

```diff
@@ -156,6 +164,7 @@ It seems that most of this boils down to:
 1. **cross-cluster migration** (how do compute and storage resources,
    and the distributed applications to which they belong, move from
    one cluster to another)
+1. cross-cluster monitoring and auditing (a.k.a. Unified visibility)
```

Comment:
Grr. Can't comment on lines that haven't changed.

Re. service discovery: Would you include multi-cluster load balancing in that category? Whether instances are spread across multiple clusters automatically, or the application is fully replicated (multi-homed) across multiple clusters manually or automatically, a common need is to spread incoming traffic across multiple clusters and to fail-over when some cluster/zone becomes unavailable.